### DISCUSSION TITLE
NORDVPN - Change to the login process

### DISCUSSION THREAD

**Discussion by onelegend** [haugene/vpn-configs-contrib#225] [General]
My container dropped out last week, then kept erroring with a SIGTERM [soft,auth-failure] received, process exiting.

I spent a while trying to diagnose this as it has been up and running for over a year - anyhow, if anyone else is getting this then NordVPN has recently disabled the username/password login system via OpenVPN as per article:
https://support.nordvpn.com/General-info/1653315162/Changes-to-the-login-process-on-third-party-apps-and-routers.htm

This change came into effect 14/06/2023

Instructions for generating the new username/password are detailed in the above article.

**Comment by ilike2burnthing** [comment]
https://github.com/haugene/docker-transmission-openvpn/issues/2638
